microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -18,7 +18,7 @@ Version 4.16 of {product-title} includes new features and enhancements. {microsh
 You can deploy {microshift-short} clusters to on-premise, cloud, disconnected, or offline environments.
 
 //TODO: Update OP system versions
-{microshift-short} {product-version} is supported on {op-system-ostree-first} and {op-system-base-full} 9.2 and 9.3.
+{microshift-short} {product-version} is supported on {op-system-base-full} and {op-system-ostree-first} and 9.4.
```

Comment:
Is it redundant to mention only MicroShift 4.16 (and above) is supported on RHEL 9.4 ? Previous versions of MicroShift are not.

Reply:
The release notes are only for 4.16. This is standard text in all of the version-specific release notes.

Reply:
Thank you for the clarification.